M2/Macaulay2/packages/Macaulay2Doc/functions/setupLift-doc.m2

```diff
@@ -0,0 +1,43 @@
+--- status: DRAFT
+--- author(s): PZJ
+--- notes: 
+
+undocumented methods setupLift
```

Comment:
Is there a reason you undocument these rather than just list the method keys as secondary keys?

Reply:
If it's because you get weird documentation errors, look at the `Synopsis` keyword used in the documentation of `preimage`, for instance.

Reply:
mostly out of laziness. Also, some of the methods are really meant for internal use only, so I need to decide whether to document those or not.

Reply:
Yeah, I would vote to keep both of these methods unexported for now (of course you can do `importFrom_Core "setupPromote"` in your own package to use it).

Reply:
When I said "some of the methods are really meant for internal use only", I did not mean that `setupPromote` and `setupLift` were for internal use only. I meant some of their methods (as listed by say `methods setupPromote`) are.

Reply:
Well you've listed two things under usage, but so they should be listed as keys.